PerfettoCds/Pipeline/DataOutput/PerfettoFrameEvent.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Microsoft.Performance.SDK;
+using System;
+using Utilities;
+
+namespace PerfettoCds.Pipeline.DataOutput
+{
+    /// <summary>
+    /// An event that represents a frame that was scheduled or actual displayed by a process. 
+    /// </summary>
+    public class PerfettoFrameEvent
+    {
+
+        public string FrameType { get; }
+        public string ProcessName { get; }
+        public long Upid { get; }
```

Comment:
Looks to be uint32 from docs - https://perfetto.dev/docs/analysis/sql-tables#expected_frame_timeline_slice